### DISCUSSION TITLE
Does this work on ARM?

### DISCUSSION THREAD

**Discussion by chookity-pokk** [swayos/swayos.github.io#1] [Q&A]
I noticed your project and it looks really cool, I was just wondering if there was anything architecture specific about it that might stop it from running on ARM devices?

**Comment by swayos** [comment]
Hi, I'm not sure about the package coverage of Arch Arm but there shouldn't be anything architecture specific. You should give it a try! :) Just install Arch Arm and launch the setup script.

**Reply by chookity-pokk** [reply]
Sweet, that would be `setup-arch.sh`?
